### PR TITLE
DOCSP-45173: Deployment & connection string

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-45173 and https://jira.mongodb.org/browse/DOCSP-45174

### Staging Links

<li><a href=https://deploy-preview-94--docs-ruby.netlify.app/get-started/create-a-connection-string>get-started/create-a-connection-string</a></li><li><a href=https://deploy-preview-94--docs-ruby.netlify.app/get-started/create-a-deployment>get-started/create-a-deployment</a></li><li><a href=https://deploy-preview-94--docs-ruby.netlify.app/get-started>get-started</a></li>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/get-started/create-a-connection-string.txt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      Paste this connection string into a file in your preferred text editor
+      and replace the ``<db_username>`` and ``<db_password>`` placeholders with
+      your database user's username and password.
+      
+      Save this file to a safe location for use in the next step.
+
+After completing these steps, you have a connection string that
+contains your database username and password.
+
+.. include:: /includes/get-started/quickstart-troubleshoot.rst
```
Comment (norareidy) [inline source/get-started/create-a-connection-string.txt:58]: Note to reviewer: this file is included in #92

File: source/get-started/create-a-connection-string.txt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ :ref:`ruby-connection-targets`.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Find your MongoDB Atlas connection string
+
+      To retrieve your connection string for the deployment that
+      you created in the :ref:`previous step <ruby-get-started-create-deployment>`,
+      log into your Atlas account and navigate to the
```
Comment (stephmarie17) [inline source/get-started/create-a-connection-string.txt:27]: [i] Per the [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#l):
```suggestion
      log in to your Atlas account and navigate to the
```

File: source/get-started/create-a-connection-string.txt [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      :guilabel:`Clusters` section. Then, click the :guilabel:`Connect` button
+      for your new deployment.
+
+      .. figure:: /includes/figures/atlas_connection_connect_cluster.png
+         :alt: The connect button in the clusters section of the Atlas UI
+
+      Proceed to the :guilabel:`Connect your application` section. Select
+      "Ruby" from the :guilabel:`Driver` selection menu and the version
+      that best matches the version you installed from the :guilabel:`Version`
+      selection menu.
```
Comment (stephmarie17) [inline source/get-started/create-a-connection-string.txt:37]: [n] Suggestion for clarity, on initial read "the version" repeating was a little hard to parse:
```suggestion
      that best matches your installed version from the :guilabel:`Version`
      selection menu.
```

File: source/get-started/create-a-connection-string.txt [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+   .. step:: Copy your connection string
+
+      Click the button on the right of the connection string to copy it to
+      your clipboard, as shown in the following screenshot:
```
Comment (stephmarie17) [inline source/get-started/create-a-connection-string.txt:42]: [s] For [accessibility purposes](https://www.mongodb.com/docs/meta/style-guide/writing/write-for-accessibility/#provide-context-for-ui-elements), maybe we can supplement the directional guidance with the name of the button. I noticed in the Atlas UI, this button is read by screenreaders as `Copy`.
```suggestion
      Click the :guilabel:`Copy` button on the right of the connection string to copy it to
      your clipboard, as shown in the following screenshot:
```

**Review by stephmarie17 — APPROVED** [review]
LGTM with a few comments ✅

File: source/get-started/create-a-connection-string.txt (hunk 4 above)
Comment (norareidy) [reply source/get-started/create-a-connection-string.txt:42]: Since the button isn't actually labeled with any text (just the copy symbol), I'm not sure if we should use :guilabel: - but I agree that just "the button on the right of the connection string" isn't ideal. I'll change to "the copy button"
